plugins/amazonq/mynah-ui/src/mynah-ui/ui/quickActions/generator.ts

```diff
@@ -21,51 +21,83 @@ export class QuickActionGenerator {
-                                          placeholder: 'Briefly describe a task or issue',
-                                          description:
-                                              'Use all project files as context for code suggestions (increases latency).',
-                                      },
-                                  ],
-                              },
-                          ]
+                            {
+                                command: '/dev',
+                                placeholder: 'Briefly describe a task or issue',
+                                description:
+                                    'Use all project files as context for code suggestions (increases latency).',
+                            },
+                        ]
                         : []),
                     ...(this.isCodeTransformEnabled
                         ? [
-                              {
-                                  commands: [
-                                      {
-                                          command: '/transform',
-                                          description: 'Transform your Java 8 or 11 Maven project to Java 17',
-                                      },
-                                  ],
-                              },
-                          ]
-                        : []),
-                    {
-                        commands: [
-                            {
-                                command: '/help',
-                                description: 'Learn more about Amazon Q',
-                            },
                             {
-                                command: '/clear',
-                                description: 'Clear this session',
+                                command: '/transform',
+                                description: 'Transform your Java 8 or 11 Maven project to Java 17',
                             },
-                        ],
+                        ]
+                        : []),
+                ],
+            },
+            {
+                commands: [
+                    {
+                        command: '/help',
+                        description: 'Learn more about Amazon Q',
+                    },
+                    {
+                        command: '/clear',
+                        description: 'Clear this session',
                     },
-                ]
+                ],
+            },
+        ]
+
+        const commandUnavailability: Record<
+            TabType,
+            {
+                description: string
+                unavailableItems: string[]
+            }
+        > = {
+            cwc: {
+                description: '',
+                unavailableItems: [],
+            },
+            featuredev: {
+                description: "This command isn't available in /dev",
+                unavailableItems: ['/dev', '/transform', '/help', '/clear'],
```

Comment:
More of a product question, but I am curious why is "/help" and "/clear" not available for feature dev?

Reply:
They didn't implement it yet, since /clear also clears the cache i believe, and the help for the /dev needs a different information in it.